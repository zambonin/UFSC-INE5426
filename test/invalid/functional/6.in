# second argument on map receives int array, expects a lambda function
int t[10], output[10]
output = map(t, t)
