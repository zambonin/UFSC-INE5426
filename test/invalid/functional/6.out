[Line 3] syntax error, unexpected ID, expecting F_LAMBDA or L_CALL
